File: copy_assets.js
import fs from 'fs';
import path from 'path';

const srcImg = path.join(process.cwd(), 'image files', 'earth.png');

if (fs.existsSync(srcImg)) {
  fs.mkdirSync(path.join(process.cwd(), 'public'), { recursive: true });
  fs.mkdirSync(path.join(process.cwd(), 'src', 'assets'), { recursive: true });
  
  fs.copyFileSync(srcImg, path.join(process.cwd(), 'public', 'earth.png'));
  fs.copyFileSync(srcImg, path.join(process.cwd(), 'src', 'assets', 'earth.png'));
  fs.copyFileSync(srcImg, path.join(process.cwd(), 'src', 'earth.png'));
  
  console.log('ASSETS_SUCCESSFULLY_COPIED');
} else {
  console.error('Source image not found:', srcImg);
}

File: copy_assets.py
import os
import shutil

src = os.path.join('image files', 'eart png.png')
dest_public = os.path.join('public', 'earth.png')
dest_assets = os.path.join('src', 'assets', 'earth.png')

os.makedirs('public', exist_ok=True)
os.makedirs(os.path.join('src', 'assets'), exist_ok=True)

if os.path.exists(src):
    shutil.copyfile(src, dest_public)
    shutil.copyfile(src, dest_assets)
    print("SUCCESS: Copied image to public/earth.png and src/assets/earth.png")
else:
    print("ERROR: Source image missing!")
